src/signals/incident-management/containers/IncidentDetail/components/AttachmentViewer/index.js

```diff
@@ -1,39 +1,69 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import styled from 'styled-components';
 
+import { Button, themeSpacing, themeColor } from '@datapunt/asc-ui';
+import { ChevronRight, ChevronLeft } from '@datapunt/asc-assets';
 import { attachmentsType } from 'shared/types';
 
-import './style.scss';
+const IconButton = styled(Button).attrs({
+  size: 32,
+  iconSize: 16,
+  variant: 'application',
+})``;
+
+const PreviousButton = styled(IconButton)`
+  position: absolute;
+  left: 10px;
+  top: calc(50% - ${themeSpacing(4)});
+`;
+
+const NextButton = styled(IconButton)`
+  position: absolute;
+  right: 10px;
+  top: calc(50% - ${themeSpacing(4)});
+`;
+
+const Wrapper = styled.div`
+  position: relative;
+  text-align: center;
+  background-color: ${themeColor('tint', 'level2')};
+  height: 722px;
```

Comment:
has this number a special meaning? If is a const could we use then a const value that has a meaningful name?

Reply:
I've used the value that was already in that component's style. I assume that that particular value has been chosen deliberately so I left it as it was.

Reply:
All right, I was thinking about something like:
```code 
const ATTACHEMENT_VIEWER_HEIGHT = 722;

...
   height: ${ATTACHEMENT_VIEWER_HEIGHT}px;
...
   max-height: ${ATTACHEMENT_VIEWER_HEIGHT}px;

```